demo/src/pages/bluesky.astro

```diff
@@ -0,0 +1,26 @@
+---
+import { BlueskyPost } from '@astro-community/astro-embed-bluesky';
+import Base from '../layouts/Base.astro';
+import post from '../post.json';
+---
+
+<Base title="BlueskyPost component example">
+	<BlueskyPost post={post.data} />
+	<p>Raw data</p>
+	<BlueskyPost post="https://bsky.app/profile/wesbos.com/post/3la2ezdodvu2g" />
+	<p>Basic</p>
+	<BlueskyPost post="https://bsky.app/profile/mk.gg/post/3la2snrsnhd2a" />
+	<p>Images</p>
+	<BlueskyPost post="https://bsky.app/profile/mk.gg/post/3l7xzeex4vf2m" />
+	<p>External link</p>
+	<BlueskyPost
+		post="https://bsky.app/profile/wattenberger.com/post/3la54isd7az2i"
+	/>
+	<p>Video</p>
+	<BlueskyPost post="https://bsky.app/profile/mk.gg/post/3l7zx4hltw62a" />
+	<p>External media with quote</p>
+	<BlueskyPost post="https://bsky.app/profile/mk.gg/post/3la4wuiii4o2e" />
```

Comment:
Ah this one too isn’t quite right I think:

![image](https://github.com/user-attachments/assets/aa176710-65ba-4a37-8719-98850046cb9a)

In the official embed the link renders inside the quoted post:

![image](https://github.com/user-attachments/assets/0c851e55-336b-4383-97e5-e8bfc7145eeb)

(The way they render the link here is weird anyway — just “github.com” — but at least the nesting helps communicate that the link is from the quoted post.)